/**:
  ros__parameters:
    debug: false
    show_pic: false
    enemy_color: 0 # 0-红色，1-蓝色
    bullet_speed: 27.0

    
    ### plane camera 1
    camera_matrix: [2055.094493, 0.000000, 664.381065,0.000000, 2051.913811, 222.698165,0.000000, 0.000000, 1.000000]
    distortion_coefficients: [-0.044477, 0.070170, -0.004976, 0.005518, 0.000000]

   

/camera_node:
    ros__parameters:
        videoflag: false
        video_path: "./Camera/new6.mp4"
        rune_exposure: 4000

/tf2_node:
    ros__parameters:
        # plane 
        odom2shooter_r: 0.078
        shooter2camera_tvec: [0.0523, 0, 0.0336]
        shooter_coordinate: "shooter"
        camera_coordinate: "camera"
        odom_coordinate: "odom"

/armor_detector_node:
    ros__parameters:
        binary_threshold_red: 50
        binary_threshold_blue: 100

/armor_tracker_node:
    ros__parameters:
        tracker.tracking_thres: 5
        tracker.lost_time_thres: 0.3
        tracker.max_v_yaw: 0.6
        tracker.flytime_offset: 0.0


/shooter_node:
    ros__parameters:
        gravity: 9.781
        mode: 115 # small:115 big:98 light:108
        k_of_small: 0.01903
        k_of_big: 0.000556
        k_of_light: 0.00053
        stop_error: 0.001
        number_of_iterations: 60
        yaw_threshold: 0.03
        pitch_threshold: 0.01

